cpp/src/table/table_device_view.cu

```diff
@@ -71,7 +71,7 @@ table_device_view_base<ColumnDeviceView, HostTableView>::table_device_view_base(
     ColumnDeviceView* d_column = _columns;
     // The beginning of the memory must be the fixed-sized ColumnDeviceView
     // objects in order for _columns to be used as an array. Therefore,
-    // any child data is assigned to the end of this array.
+    // any child data is assigned to the end of this array (h_end/d_end).
     int8_t* h_end = (int8_t*)(h_column + source_view.num_columns());
     int8_t* d_end = (int8_t*)(d_column + source_view.num_columns());
```

Comment:
SUGGESTION:
Same here, auto d_end = reinterpret_cast<int8_t*>(d_column + source_view.num_columns()) is more modern and also more consistent with other places in cuDF.